doc/rules/control_structure/yoda_style.rst

```diff
@@ -2,8 +2,9 @@
 Rule ``yoda_style``
 ===================
 
-Write conditions in Yoda style (``true``), non-Yoda style (``false``) or ignore
-those conditions (``null``) based on configuration.
+Enforce Yoda style conditions (``true``), disable Yoda style conditions
```

Comment:
`disable Yoda style` sounds like we don't enforce it
while
`non-Yoda style` sounds like revert os Yoda style

the original wording sounds more accurate to me, what do you think ?

Reply:
I would take the original one, but replace `false` with `['equal' => false, 'identical' => false, 'less_and_greater' => false]` to avaoid [confusion](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5281).

Reply:
@kubawerlos I agree that's a clear expression. Let me change the PR to that.

Reply:
Done https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5293/commits/1d4b127a2245a9b4151acc1953a14d2a0fdada67